pkgs/development/libraries/timezonemap/default.nix

```
@@ -0,0 +1,65 @@
+{ stdenv
+, autoreconfHook
+, fetchbzr
+, pkgconfig
+, gtk3
+, glib
+, file
+, gobject-introspection
+, json_glib
```

```suggestion
, json-glib
```

`json_glib` is an alias to `json-glib`.